package.json

```diff
@@ -7,6 +7,16 @@
   "es2015": "./dist/esm/index.js",
   "types": "./dist/types/index.d.ts",
   "sideEffects": false,
+  "exports": {
+    ".": {
+      "node": "./dist/cjs/index.js",
+      "default": "./dist/esm5/index.js"
+    },
+    "./operators": {
+      "node": "./dist/cjs/operators/index.js",
+      "default": "./dist/dist/esm5/operators/index.js"
+    }
```

Comment:
There would also need to be entries for:

- rxjs/testing
- rxjs/webSocket
- rxjs/fetch
- rxjs/ajax